### PR TITLE
Add support for --topology argument in jenkins

### PR DESCRIPTION
**Pull request by cescgina**
Modify the get_deployment method of the jenkins source to get topology                                                                                                                    
information. The information is obtained from the job name or from                                                                                                                        
artifacts. If many jobs are requested, it falls back to parsing the job                                                                                                                   
name. If few, it request some artifacts from the last build and tries to                                                                                                                  
find the information there. This is not completely robust, since                                                                                                                          
different jobs have different artifacts. If a job has no last build or                                                                                                                    
the artifacts used are not found, it will also fallback to extracting                                                                                                                     
the information from the jobs name.

### CODE REVIEW

**Comment by cescgina** [comment]
Depends on #121, once that is merged I will rebase and update the changes.

File: cibyl/plugins/openstack/deployment.py [hunk 1]
```diff
@@ -68,8 +84,15 @@ def __str__(self, indent=0, verbosity=0):
...
         if self.services:
-            info += f'\n{indent_space}  ' + Colors.blue('Service: ')
+            info += f'\n{indent_space}' + Colors.blue('Service: ')
             info += f'{self.services.value}'
+        if self.ip_version:
+            info += f'\n{indent_space}' + Colors.blue('IP version: ')
+            info += f'{self.ip_version}'
+        if self.topology:
+            info += f'\n{indent_space}' + Colors.blue('Topology: ')
+            info += f'{self.topology}'
```
[the remark this answers is not in the document]
Comment (cescgina) [reply cibyl/plugins/openstack/deployment.py:97]: That's a good point, I'll add a check for it

[the remark this answers is not in the document]
Comment (cescgina) [reply cibyl/plugins/openstack/deployment.py:97]: Done, now it'll only be printed if the value is populated

**Review by adrianfusco — APPROVED** [review]
Good job Joan. LGTM!

**Pull request merged**